src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -178,7 +182,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             DockerAccess access = null;
             try {
                 access = createDockerAccess(minimalApiVersion);
-                ServiceHub serviceHub = serviceHubFactory.createServiceHub(project, session, access, log, logSpecFactory);
+                fetchLimit = resolvedImages.get(0).getRunConfiguration().getFetchLimit() != null
```

Comment:
I'm also only on 1.21, however there is `since` query param (not as part of the filter), but it seems to eclusive, not inclusive (so the entry with the name itself is not included). 

For the name lookup I have to check why I haven't use the `/inspect` call. I remember darkly, there was a reason ;-)

'have to run now, see you tomorrow ...